templates/cdk-erigon/config.yml

```diff
@@ -238,7 +238,9 @@ zkevm.data-stream-port: {{.zkevm_data_stream_port}}
 # - 2: BigEndian (improved format)
 # - 3: Latest format with additional features
 # Default: 2
-# fork12 or after {{if eq .zkevm_rollup_fork_name "banana"}}
+# {{if eq .cdk_erigon_node_image "hermeznetwork/cdk-erigon:v2.63.0-pp2-beta1"}}
```

Comment:
we will probably need to consider how to adapt this to new versions, but for now, it is fine.